common/persistence/visibility/store/sql/query_converter_sqlite.go

```diff
@@ -336,8 +337,15 @@ func (c *sqliteQueryConverter) buildCountStmt(
 		whereClauses = append(whereClauses, queryString)
 	}
 
+	groupByClause := ""
+	if len(groupBy) > 0 {
+		groupByClause = fmt.Sprintf("GROUP BY %s", strings.Join(groupBy, ", "))
+	}
+
 	return fmt.Sprintf(
-		"SELECT COUNT(1) FROM executions_visibility WHERE %s",
+		"SELECT %s FROM executions_visibility WHERE %s %s",
+		strings.Join(append(groupBy, "COUNT(1)"), ", "),
```

Comment:
Isn't `COUNT(*)` is more canonical way to do it?

Reply:
Hm, I don't know. I can replace with `COUNT(*)`.